torch_glow/src/PyTorchCommon.cpp

```diff
@@ -107,8 +107,10 @@ glow::ElemKind scalarTypeToElemKind(c10::ScalarType ty) {
     return ElemKind::Int64ITy;
   } else if (ty == at::kBool) {
     return ElemKind::BoolTy;
+  } else if (ty == at::kQInt8) {
```

Comment:
should we just add the uint8 and int32 while we're at it?

Reply:
yeah agree.